R/data-mlbbat10.R

```diff
@@ -30,58 +30,56 @@
 #' @source \url{https://www.mlb.com}, retrieved 2011-04-22.
 #' @keywords datasets
 #' @examples
-#' \dontrun{
+#' library(ggplot2)
+#' library(dplyr)
+#' library(scales)
 #' d   <- mlbbat10[mlbbat10$at_bat > 200,]
 #' pos <- list(c("OF"), c("1B", "2B", "3B", "SS"), "DH", "C")
 #' POS <- c("OF", "IF", "DH", "C")
 #'
 #' #=====> On-base Percentage Across Positions <=====#
-#' out <- c()
-#' gp  <- c()
-#' for(i in 1:length(pos)){
-#' 	these <- which(d$position %in% pos[[i]])
-#' 	out   <- c(out, d[these, "obp"])
-#' 	gp    <- c(gp, rep(POS[i], length(these)))
-#' }
-#' plot(out ~ as.factor(gp))
-#' summary(lm(out ~ as.factor(gp)))
-#' anova(lm(out ~ as.factor(gp)))
+#' dplot1 <- d %>%
+#'   select(position, obp)
+#' dplot1 %>%
+#'   ggplot(aes(fill = position, x = position, y = obp)) +
+#'     geom_boxplot() +
+#'     labs(title = "On-base Percentage Across Positions", fill = NULL,
+#'                   y = "On-base Percentage", x = "Position") +
```

Comment:
title, fill, x, and y should each be on their own lines, indented from `labs()`. More natural orders would be x, y, fill, title or title, x, y, fill.